setup.py

```diff
@@ -61,17 +64,18 @@ def get_requirements():
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/openvinotoolkit/datumaro",
-    packages=setuptools.find_packages(exclude=['tests*']),
+    packages=setuptools.find_packages(include=['datumaro']),
     classifiers=[
         "Programming Language :: Python :: 3",
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
     ],
     python_requires='>=3.6',
-    install_requires=get_requirements(),
+    install_requires=get_requirements(CORE_REQUIREMENTS_FILE),
     extras_require={
         'tf': ['tensorflow'],
         'tf-gpu': ['tensorflow-gpu'],
+        'default': get_requirements(DEFAULT_REQUIREMENTS_FILE),
```

Comment:
The result of `get_requirements` includes the OpenCV dependency, which should not be included here.

Reply:
Fixed